testsuite/integration-tests/pom.xml

```diff
@@ -181,6 +181,28 @@
                 </plugins>
             </build>
         </profile>
+        <!-- 
+            Test against MicroProfile config.
+            standalone-microprofile.xml is used to replace occurrences of
+            standalone-full.xml in Arquillian configuration (arquillian.xml)
+        -->
+        <profile>
+            <id>standalone.microprofile.profile</id>
+            <activation>
+                <property>
+                    <name>ts.standalone.microprofile</name>
+                </property>
+            </activation>
+            <properties>
+                <jboss.server.config.file.name>standalone-microprofile.xml</jboss.server.config.file.name>
+                <!--
+                    Excluding those tests that would need modules 
+                    which are in standalone-full.xml but not in 
+                    standalone-microprofile.xml
+                -->
+                <additional.surefire.excluded.groups>org.jboss.resteasy.category.ExpectedFailingWithStandaloneMicroprofileConfiguration,org.jboss.resteasy.category.ExpectedFailingBecauseOfSmallRyeMicroprofileOpenApi11</additional.surefire.excluded.groups>
```

Comment:
ExpectedFailingBecauseOfSmallRyeMicroprofileOpenApi11 issue needs to be investigated.